api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -121,7 +121,7 @@ private CdrVersionTier getVersionsForTier(DbAccessTier accessTier) {
   private DbCdrVersion getDefaultVersionForTier(
       DbAccessTier accessTier, List<DbCdrVersion> cdrVersions) {
     List<DbCdrVersion> defaultVersions =
-        cdrVersions.stream().filter(DbCdrVersion::getIsDefaultNotNull).collect(Collectors.toList());
+        cdrVersions.stream().filter(DbCdrVersion::getIsDefault).collect(Collectors.toList());
```

Comment:
renamed the non-nullable version to `getIsDefault()` and the nullable to `getIsDefaultNullable()`